docs/channel.md

```diff
@@ -2,81 +2,64 @@
 
 # Channels
 
-Nextflow is based on the dataflow programming model in which processes communicate through channels.
+In Nextflow, **channels** are the key data structure used to facilitate the dataflow dependencies between each step (i.e. {ref}`process <process-page>`) in a pipeline.
 
-A channel has two major properties:
+There are two kinds of channels, *queue channels* and *value channels*. Channels are created using *channel factories* and transformed using *channel operators*.
 
-1. Sending a message is an *asynchronous* (i.e. non-blocking) operation, which means the sender doesn't have to wait for the receiving process.
-2. Receiving a message is a *synchronous* (i.e. blocking) operation, which means the receiving process must wait until a message has arrived.
-
-(channel-types)=
+(channel-type-queue)=
 
-## Channel types
+## Queue channels
 
-In Nextflow there are two kinds of channels: *queue channels* and *value channels*.
+A *queue channel* is a channel that *emits* an asynchronous sequence of values.
 
-(channel-type-queue)=
+A queue channel can be created by channel factories ({ref}`channel.of <channel-of>`, {ref}`channel.fromPath <channel-path>`, etc), operators ({ref}`operator-map`, {ref}`operator-filter`, etc), and processes (see {ref}`Process outputs <process-output>`).
```

Comment:
```suggestion
A queue channel can be created by channel factories (e.g., {ref}`channel.of <channel-of>` and {ref}`channel.fromPath <channel-path>`), operators (e.g., {ref}`operator-map` and  {ref}`operator-filter`), and processes (see {ref}`Process outputs <process-output>`).
```

Reply:
Ect is unclear because it's not established which channel factories can create queue channels.

Reply:
Agreed, thanks